docs/releases/v1.0.1.md

```diff
@@ -1,13 +0,0 @@
-# Release notes
```

Comment:
why all the old release notes are deleted?

Reply:
Can you please add also the release notes for the changes introduced by this PR? use one of the previous as an example.

I understand there are breaking changes with this update, in that case, the file should be named `v2.0.0.md`, 
If the changes are not breaking we can use `v1.3.0.md` instead.

Thank you!